TreeMaker/python/doLostLeptonBkg.py

```diff
@@ -8,18 +8,23 @@ def doLostLeptonBkg(self,process,METTag):
             pfCandsTag  = cms.InputTag('packedPFCandidates')
         )
 
+    from TreeMaker.TreeMaker.TMEras import TMeras
     from TreeMaker.Utils.isolationproducer_cfi import isolationproducer
     process.IDMuonMiniIso = isolationproducer.clone(
         LeptonTag = cms.InputTag('LeptonsNew:IdMuon'),
-        LeptonType = cms.string('muon'),

```

Comment:
the leptontype values must be specified here

Reply:
I'm not sure what you mean. In the code I'm looking at there is a line which specifies the LeptonType. Did you maybe comment on the old code?

Reply:
ah, nvm